test/opsem/gstr.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: %seabmc "%s" 2>&1 | %oc %s
+; CHECK: ^unsat$
+; ModuleID = '/tmp/sea-4rYJtO/gstr.pp.ms.bc'
+source_filename = "test-globals/gstr.c"
+target datalayout = "e-m:e-p:32:32-f64:32:64-f80:32-n8:16:32-S128"
+target triple = "i386-pc-linux-gnu"
+
+@.str = private unnamed_addr constant [4 x i8] c"abc\00", align 1
+@llvm.used = appending global [4 x i8*] [i8* bitcast (void ()* @seahorn.fail to i8*), i8* bitcast (void (i1)* @verifier.assume to i8*), i8* bitcast (void (i1)* @verifier.assume.not to i8*), i8* bitcast (void ()* @verifier.error to i8*)], section "llvm.metadata"
+
+declare i32 @unknown1(...) local_unnamed_addr #0
+
+declare void @verifier.assume(i1)
+
+declare void @verifier.assume.not(i1)
+
+declare void @seahorn.fail()
+
+; Function Attrs: noreturn
+declare void @verifier.error() #1
+
+declare void @seahorn.fn.enter() local_unnamed_addr
+
+; Function Attrs: nounwind
+define i32 @main() local_unnamed_addr #2 {
+entry:
+  tail call void @seahorn.fn.enter() #3
+  %0 = tail call i32 bitcast (i32 (...)* @unknown1 to i32 ()*)() #3
+  %1 = icmp eq i32 %0, 0
+  br i1 %1, label %verifier.error, label %.lr.ph
+
+.lr.ph:                                           ; preds = %entry
+  br label %2
+
+; <label>:2:                                      ; preds = %.lr.ph, %2
+  %.0.i1 = phi i32 [ 1, %.lr.ph ], [ %3, %2 ]
+  %3 = add nsw i32 %.0.i1, 2
+  %4 = trunc i32 %3 to i8
+  store i8 %4, i8* getelementptr inbounds ([4 x i8], [4 x i8]* @.str, i32 0, i32 2), align 1, !tbaa !3
+  %5 = tail call i32 bitcast (i32 (...)* @unknown1 to i32 ()*)() #3
+  %6 = icmp eq i32 %5, 0
+  br i1 %6, label %verifier.error.loopexit, label %2
+
+verifier.error.loopexit:                          ; preds = %2
+  br label %verifier.error
+
+verifier.error:                                   ; preds = %verifier.error.loopexit, %entry
+  tail call void @verifier.assume.not(i1 true) #3
```

Comment:
the assumption got compiled to `true`. This test will return `unsat` independently of the other instructions.